cisco_secure_web_appliance/README.md

```diff
@@ -0,0 +1,274 @@
+Click on Commit Changes at top right-side of Log Subscriptions Page.
+
+    **NOTE:** These changes will not go into effect until you commit them.
+
+#### Steps to set time zone to GMT in Cisco Secure Web Appliance 
+
+Since Datadog expects all the logs in GMT timezone by default, if the time zone of your Cisco Secure Web Appliance logs is other than GMT, please change it to GMT. Here are the steps:
+1. Go to System Administration->Time Zone.
+2. Click on `Edit Settings`.
+3. Select GMT Offset as the region.
+4. Select GMT as the Country.
+5. Select GMT (GMT) as the time zone.
+6. Submit and commit the changes.
+
+
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `cisco-secure-web-appliance` under the Checks section.
+
+## Data Collected
+
+### Log 
+
+| Format     | Event Types    |
+| ---------  | -------------- |
+| syslog | access_logs, l4tm_logs |
+
+### Metrics
+
+The Cisco Secure Web Appliance does not include any metrics.
+
+### Events
+
+The Cisco Secure Web Appliance integration does not include any events.
+
+### Service Checks
+
+The Cisco Secure Web Appliance integration does not include any service checks.
+
+## Troubleshooting
+
+**Permission denied while port binding:**
+
+If you see a **Permission denied** error while port binding in the Agent logs, see the following instructions:
+
+1. Binding to a port number under 1024 requires elevated permissions. Grant access to the port using the `setcap` command:
+    ```shell
+    sudo setcap CAP_NET_BIND_SERVICE=+ep /opt/datadog-agent/bin/agent/agent
+    ```
+
+2. Verify the setup is correct by running the `getcap` command:
+
+    ```shell
+    sudo getcap /opt/datadog-agent/bin/agent/agent
+    ```
+
+    With the expected output:
+
+    ```shell
+    /opt/datadog-agent/bin/agent/agent = cap_net_bind_service+ep
+    ```
+
+    **Note**: Re-run this `setcap` command every time you upgrade the Agent.
+3. [Restart the Agent][3].
+
+
+**Permission denied while file monitoring:**
+
+If you see a **Permission denied** error while monitoring the log files, give the `dd-agent` user read permission on them.
+  ```shell
+    sudo chmod g+s Path/to/Directory/Where/Logs/would/Get/Stored/
+  ```
+  ```shell
+    sudo chgrp dd-agent Path/to/Directory/Where/Logs/would/Get/Stored/
+  ```
+
+
+**Data is not being collected:**
+
+Make sure that traffic is bypassed from the configured port if the firewall is enabled.
+
+**Port already in use:**
+
+- If you see the **Port \<PORT-NO> Already in Use** error, see the following instructions. The example below is for PORT-NO = 514:
+
+- On systems using Syslog, if the Agent listens for access_logs logs on port 514, the following error can appear in the Agent logs: `Can't start UDP forwarder on port 514: listen udp :514: bind: address already in use`.
+
+- This error occurs because by default, Syslog listens on port 514. To resolve this error, take **one** of the following steps: 
```

Comment:
```suggestion
If you see the **Port \<PORT-NO> Already in Use** error, see the following instructions. The example below is for PORT-NO = 514:

On systems using Syslog, if the Agent listens for access_logs logs on port 514, the following error can appear in the Agent logs: `Can't start UDP forwarder on port 514: listen udp :514: bind: address already in use`.

This error occurs because by default, Syslog listens on port 514. To resolve this error, take **one** of the following steps: 
```

Reply:
Updated as per the suggestion.